AzurePullRequests.Restore/Enums/RestoreType.cs

```diff
@@ -1,10 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
```

Comment:
Comment in code with attachment
![images](https://github.com/user-attachments/assets/73fe4a8c-dc41-46e1-afcd-b93145a47a29)